tower-hedge/src/rotating.rs

```diff
@@ -0,0 +1,76 @@
+extern crate tokio_timer;
+
+use std::time::{Duration, Instant};
+
+use tokio_timer::clock;
+
+/// This represents a "rotating" value which stores two T values, one which
+/// should be read and one which should be written to.  Every period, the
+/// read T is discarded and replaced by the write T.  The idea here is that
+/// the read T should always contain a full period (the previous period) of
+/// write operations.
+pub struct Rotating<T> {
```

Comment:
It seems like more effort than it is worth to make this generic over `T` as the only thing it is every used for is `Histogram<u64>`?